### DISCUSSION TITLE
Package name unchanged?

### DISCUSSION THREAD

**Discussion by allenluce** [bromite/bromite#1600] [Builds]
Issue #1 says that Bromite now builds with its own package name (ostensibly `org.bromite.chromium`). When I build Chromium with the full set of patches, I get a package name of `org.chromium.chrome.stable`. I don't see anything in the repo (from 19bcbb3) that would appear to actually change the package name.

What am I missing?

**Comment by uazo** [comment]
> What am I missing?

nothing.
it is a specific will of @csagan5 to not want to show the final patch that transforms chromium into bromite, which although it may not be appreciated, is intended in order not to have unauthorized dangerous clones.

**Reply by csagan5** [reply]
> is intended in order not to have unauthorized dangerous clones

That can still happen; the intent is simply to make it a bit harder and avoid support requests about "I cannot update Bromite" because someone installed a colliding package name version with unofficial signatures.

**Comment by csagan5** [comment]
The package name change and icon are not included, on purpose; give it any name you like (and you do not have anyways the keys to sign it as `org.bromite.chromium`).
